data/data/gcp/bootstrap/main.tf

```diff
@@ -21,7 +21,10 @@ data "ignition_config" "redirect" {
 }
 
 resource "google_compute_address" "bootstrap" {
-  name = "${var.cluster_id}-bootstrap-public-ip"
+  name = "${var.cluster_id}-bootstrap-ip"
+
+  address_type = var.public_endpoints ? "EXTERNAL" : "INTERNAL"
```

Comment:
we are missing the make sure bootstrap is only allowed SSH from network https://github.com/openshift/installer/blob/123f41e4dacf26e46dd0660f819841f5d14cb315/data/data/gcp/bootstrap/main.tf#L30-L41

Reply:
Done. https://github.com/openshift/installer/pull/2522/files#diff-86f47745cfad644d2117fcbce91f8b92L36